core/store/src/main/java/uk/gov/gchq/gaffer/store/serialiser/lengthvalue/EdgeIdSerialiser.java

```diff
@@ -89,4 +89,9 @@ public EdgeId deserialiseEmpty() throws SerialisationException {
     public boolean preservesObjectOrdering() {
         return false;
     }
+
+    @Override
+    public boolean isConsistent() {
+        return true;
```

Comment:
This should actually be:
return null != vertexSerialiser && vertexSerialiser.isConsistent();